spec.bs

```diff
@@ -364,15 +455,29 @@ To get the <dfn>max batch size</dfn> for an [=/origin=] |issuer|, run the follow
 
 To <dfn>generate masked tokens</dfn> given [=key commitments=] |issuerKeys| and number |numTokens|, run the following steps:
 
-1. Let |maskOperation| be the Mask operation corresponding to |issuerKeys|["protocol_version"].
-1. Let |result| ([=tuple=] ([=byte sequence=], [=byte sequence=])) be the result of running |maskOperation| on |issuerKeys| and |numTokens|.
-1. If |result| is null, return an error.
-1. Return |result|.
-
-To <dfn>unmask tokens</dfn> given [=key commitments=] |issuerKeys|, byte string |pretokens|, and a bytestring |response|, run the following steps:
-
-1. Let |unmaskOperation| be the Unmask operation corresponding to |issuerKeys|["protocol_version"].
-1. Let |result| ([=list=] of byte strings) be the result of running |unmaskOperation| on |pretokens| and |response|.
+1. Let |issueRequest| be an empty [=IssueRequest=].
+1. Set |issueRequest|["count"] to |numTokens|.
+1. Let |blinds| be an empty [=list=].
+1. Repeat the following steps, |numTokens| times:
+    1. Let |input| be a random [=byte sequence=].
+    1. Let (|blind|, |blindedElement|) ([=tuple=] ([=byte sequence=], [=byte sequence=])) be the result of running the [=Blind=] function of the [[!VOPRF]] protocol on |input|, where |blindedElement| is encoded as a X9.62 uncompressed point.
+    1. [=list/Append=] the |blind| to |blinds|.
+    1. Append the second element of |blindedElement| to |issueRequest|["nonces"].
+1. Let |issueHeader| be the serialization of |issueRequest|.
```

Comment:
Refer to serialization method spec? I assume this is in W3C realm and there is a spec for this.

Reply:
Its treat issueRequest as a byte string (using the definition of an IssueRequest above, not sure if there's a better way to say this?